typo fix: unlinke --> unlike (#377)
devopshq/artifactory: CHANGELOG.md

- Add docs about `stat()`

## DevOpsHQ (dohq-artifactory) => Original (Parallels)

> Our library `dohq-artifactory` have this diff (unlike the original)

Reviewer: ```suggestion
`dohq-artifactory` library has this diff (unlike the original)
```
